spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/onnx/Constant.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.intel.analytics.bigdl.nn.onnx
+
+import scala.reflect.ClassTag
+// import com.intel.analytics.bigdl.nn.abstractnn.TensorModule
```

Comment:
Code clean.

Reply:
@i8run Thanks for your time.
According to your suggestions
1. changed parameter's name
2. removed some unwanted layers, added forward tests for the remainning layers, and those layers do not have backward, e.g. Shape layer, it only depends on the input, gradient does not apply to it, right?
3. cleaned up code.